Source/Core/VideoCommon/RenderBase.h

```diff
@@ -79,48 +79,52 @@ class Renderer
   virtual void RestoreState() {}
   virtual void ResetAPIState() {}
   virtual void RestoreAPIState() {}
+  // Some of the methods called by here assume g_renderer is initialized, therefore
```

Comment:
This should probably have a TODO or FIXME on it. This is very indicative of an initialization order issue or dependencies being where they shouldn't be.